src/components/lyne-panel/lyne-panel.scss

```diff
@@ -0,0 +1,93 @@
+:host {
+  display: block;
+
+  --font-family: 'SBBWeb Light';
+  --letter-spacing: 0.01em;
```

Comment:
We should probably add a TODO here to use the CSS Variables, once the typo styles are up to date again.

Reply:
done